maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -175,6 +175,8 @@ public class MavenCli {
 
     private CLIManager cliManager;
 
+    private static final Pattern NEXT_LINE = Pattern.compile("(\r\n)|(\r)|(\n)");
```

Comment:
The capturing groups are redudant and we likely don't care about Mac classic LS. `\r?\n` is more than enough.

Reply:
DOS & Windows: \r\n 0D0A (hex), 13,10 (decimal)
Unix & Mac OS X: \n, 0A, 10
Macintosh (OS 9): \r, 0D, 13

so your pattern not applicable to OS 9 and similar

Reply:
OS 9 is dead for 20+ years.